modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -0,0 +1,26 @@
+== Inline formatting.
+
+List bullets and list numbers can new be inline-formatted.
+
+Inline-formatting is the direct formatting of an object. It contrasts with formatting applied to an object because that object is a particular type of object.
+
+If all Table Headers are formatted a particular way, that is not inline formatting. When a particular Table Header is selected and has formatting specifically applied to it, that is inline formatting.
+
+Inline-formatting is also known as spot-formatting.
```

Comment:
I don't think this and the above two paragraphs are relevant to this. I always find that getting to the point is the best approach

Reply:
I’ve re-written and re-organised the explanatory text.